umake/tools.py

```diff
@@ -337,6 +337,15 @@ def create_launcher(desktop_filename, content):
         gsettings.set_strv("favorites", launcher_list)
 
 
+def add_exec_link(exec_path, destination_name):
```

Comment:
Can I override the value for the binary directory defined in settings.py? I need it to point to the tmpfolder created in the small test